### ISSUE TITLE
Investigate how recurring only fees can be removed from the initial invoice

### ISSUE THREAD

**Issue by james-allan** [Automattic/woocommerce-payments#2972]
### Describe the bug

When using the `woocommerce_cart_calculate_fees` hook to add recurring only fees, the fees get added to the initial Stripe Billing invoice but shouldn't as it doesn't apply to the initial order in WC. 

### To Reproduce

1. Add any subscription product to the cart.
2. Add the following code snippet to your site.
3. Place the order using WC Payments.
4. Check the Stripe billing subscription in the Stripe dashboard. 
5. Go to the subscription's first invoice. 
6. Note that the invoice contains the custom fees. 

```php
add_filter( 'woocommerce_cart_calculate_fees', 'add_recurring_postage_fees', 10, 1 );

function add_recurring_postage_fees( $cart ) {
    if ( ! empty( $cart->recurring_cart_key ) ) {
        $cart->add_fee( 'Postage', 5 );
	$cart->add_fee( 'Handling', 15 );
    }
}
```

### Actual behavior

The recurring only fees are added to the initial invoice because of a limitation with Stripe billing and how the first invoice is created. See [this comment](url) for more details. 

### Screenshots

<img width="467" alt="Screen Shot 2021-09-23 at 10 07 22 am" src="https://user-images.githubusercontent.com/8490476/134437870-8fdceb4d-3e19-45f4-b761-961cb15f21d5.png">

_Fees in the recurring cart but not the initial order_

<img width="1429" alt="Screenshot on 2021-09-20 at 14-59-14" src="https://user-images.githubusercontent.com/8490476/134437660-e77510d1-cd29-4b2f-95ba-ffa0fcde7c0c.png">

_Fees on the first invoice in Stripe_

<img width="2075" alt="Screenshot on 2021-09-20 at 15-00-38" src="https://user-images.githubusercontent.com/8490476/134437690-852f1425-ec6b-4bab-a3a0-a67c55bb8bfb.png">

_No fees on the first order_

### Expected behavior

The initial invoice in Stripe should reflect the first order in WC. However this may just be a reporting issue given the first invoice isn't charged. Payment for the first order in WC is taken via WC Payments.

### Additional context

Recurring only fees are somewhat rare. Third-party plugins and code can add them to the initial payment, or two specific recurring carts, or to all cart objects. How they can be used with Subscriptions is outlined in our documentation here: https://docs.woocommerce.com/document/subscriptions/develop/recurring-cart-fees/

**Comment by haszari** [comment]
@james-allan Is this issue still relevant? If so, can you update the description with simple steps to demonstrate the bug in a shopper or merchant flow. If not - feel free to close :)

**Comment by james-allan** [comment]
I'm going to close this one. The issue only affects fees that are added to the initial payment, but not the subscription. eg a custom admin fee that is charged at sign up but not on subscription renewals. 

In those cases, the initial invoice in Stripe doesn't match the order in WC, however since this issue was created, there have been a number of scenarios where the initial invoice isn't a reflection of the order in WC. We have also discovered that this isn't a problem as the initial invoice in Stripe is created in order to create the subscription and isn't (and can never be) a reference to an actual order in WC.  

I tested the other fee scenarios (fees applied to both the initial charge and subscription renewals, and only to the subscription) and both of them work fine.
